src/cli/cli.cpp

```diff
@@ -5700,6 +5774,35 @@ template <> otError Interpreter::Process<Cmd("panid")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli parent
+ * @code
+ * parent
+ * Ext Addr: be1857c6c21dce55
+ * Rloc: 5c00
+ * Link Quality In: 3
+ * Link Quality Out: 3
+ * Age: 20
+ * Version: 4
+ * Done
+ * @endcode
+ * @code
+ * CSL clock accuracy: 20
```

Comment:
This should be part of the example above, right?  So remove the endcode / code on lines 5788 & 5789